adm/systemv/beanstalkd.init

```diff
@@ -0,0 +1,139 @@
+#!/bin/sh
+#
+# System V init script in charge of starting/stopping beanstalkd
+#
+# chkconfig: - 57 47
+# description: beanstalkd is a simple, fast work queue
+# processname: beanstalkd
+# config: /etc/sysconfig/beanstalkd
+# pidfile: /var/run/beanstalkd/beanstalkd.pid
+
+### BEGIN INIT INFO
+# Provides: beanstalkd
+# Required-Start: $local_fs $network $remote_fs
+# Required-Stop: $local_fs $network $remote_fs
+# Default-Stop: 0 1 2 6
+# Short-Description: start and stop beanstalkd
+# Description: beanstalkd is a simple, fast work queue
+### END INIT INFO
+
+# Source function library
+. /etc/rc.d/init.d/functions
+
+# Source networking configuration
+. /etc/sysconfig/network
+
+# Check that networking is up
+[ "$NETWORKING" = "no" ] && exit
+
+exec="/usr/bin/beanstalkd"
+prog=$(basename $exec)
+
+# Default options, overruled by items in sysconfig
+BEANSTALKD_ADDR=127.0.0.1
+BEANSTALKD_PORT=11300
+BEANSTALKD_USER=beanstalkd
+
+[ -e /etc/sysconfig/${prog} ] && . /etc/sysconfig/${prog}
+
+lockfile=/var/lock/subsys/${prog}
+
+start() {
+	[ -x $exec ] || exit 5
+	echo -n $"Starting $prog: "
+
+	options="-l ${BEANSTALKD_ADDR} -p ${BEANSTALKD_PORT} -u ${BEANSTALKD_USER}"
+
+	if [ "${BEANSTALKD_MAX_JOB_SIZE}" != ""  ]; then
```

Comment:
I would have preferred:

```
if [ -n "$BEANSTALKD_MAX_JOB_SIZE" ]; then
```

instead...

Reply:
We still can fix it. Feel free to make a PR. I am not very proficient in bash, truth to be told. My level is only average.